packages/fxa-content-server/grunttasks/copy.js

```diff
@@ -80,7 +80,7 @@ module.exports = function (grunt) {
       dest: '<%= yeoman.tmp %>/styles/',
       dot: true,
       expand: true,
-      src: '{,*/}*.css',
+      src: ['main.css', 'tailwind.out.css'],
```

Comment:
All this change does for us is that it prevents copying `tailwind.css` into the `.tmp` directory on build. We don't need that file, just `tailwind.out.css`.